package-build.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: kctrl.carvel.dev/v1alpha1
+kind: PackageBuild
+metadata:
+  name: secretgen-controller.carvel.dev
+spec:
+  release:
+  - resource: {}
+  template:
+    spec:
+      app:
+        spec:
+          deploy:
+          - kapp: {}
+          template:
+          - ytt:
+              paths:
+              - config/package-bundle/config
+              - tmp/output-config.yml
```

Comment:
`tmp` folder in the imgpkg bundle doesn't sound right. Can we rename it?

Reply:
yeah now putting it inside config folder only.